File: ch2/2-10.go
package main

import "fmt"

func main() {
	s := "JonaTHan"
	fmt.Printf("%s\n", s)
	for i := 0; i < 8; i++ {
		fmt.Printf("%c", lower(rune(s[i])))
	}
	fmt.Println()
}

func lower(c rune) rune {
	if c >= 'A' && c <= 'Z' {
		return c + 'a' - 'A'
	}
	return c
}

File: kr/ch2/2-10.c
#include <stdio.h>
#include <string.h>

int lower(int c);

int main(void)
{
    char *s = "Jonathan Torres";
    printf("%s\n", s);

    for (size_t i = 0; i < strlen(s); i++) {
        putchar(lower(s[i]));
    }

    printf("\n");
}

int lower(int c)
{
    return (c >= 'A' && c <= 'Z') ? c + 'a' - 'A' : c;
}
